scripts/testing/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "2"
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/${NAMESPACE:-elasticsearch}/elasticsearch:${ES_VERSION}
+    environment:
+      - "discovery.type=single-node"
+      - "ES_JAVA_OPTS=-Xms1024m -Xmx1024m"
```

Comment:
Is 1GB enough? I'd make it 2 - it's our setup for regular ftests

Reply:
In this case, we will need to change the default VM type. I assume some of the tests will not require 2GBs so we should probably make it configurable. Or another idea, we can group heavy tests in run them on a bigger VM. 

Anyway, it's too early to adjust it because we don't have many test cases.